source/extensionPoints/util.py

```diff
@@ -7,6 +7,7 @@
 used, however for more advanced requirements these utilities can be used directly.
 """
 
+from __future__ import annotations

```

Comment:
what is this needed for?

Reply:
it treats annotations as strings. In these, we need this to reference BoundMethodWeakref in one of the init params of itself.